### ISSUE TITLE
Bad Interpreter Error

### ISSUE THREAD

**Issue by crisron-zz** [MayankPratap/show-down#19]
I downloaded the script and made it executable. On executing the script `./showDown.py`, I got the following error message: 

``` bash
    -bash: ./showDown.py: /usr/bin/python^M: bad interpreter: No such file or directory
```

Please see to it.

**Comment by kartikayx** [comment]
Please check the script shebang you might have edited it incorrectly ,
it must be 
`#!/usr/bin/python`
in your case it seems to be 
`#!/usr/bin/python^M`
the extra `^M` might be causing the error
Please let me know if it helps ...

**Comment by kartikayx** [comment]
Some issues has been fixed , try to download latest version of script .. :)
This might solve your problem

**Comment by kartikayx** [comment]
closed #19
